### DISCUSSION TITLE
Tensorrt detector vs CPU

### DISCUSSION THREAD

**Discussion by bimmbo** [blakeblackshear/frigate#5388] [General]
Does Tensorrt object detection use a different model/dataset than the standard CPU detector?  I recently updated to the betas using an Nvidia Quadro P400 as my detector and have noticed that my dogs are no longer picked up in the events.  I looked in the debugging viewer and the dogs are picked up as objects in motion but never identified (occasionally IDed as persons).  If i fire up version 0.11 with only a cpu detector, dogs are picked up instantly.

With the beta, persons, cars, boats are all picked up no problem.  I have tried use yolov4-tiny-416 and yolo47-tiny-416 with no difference in object detection noticed.  

I can't figure out why using the tensorrt detector seems to be worse at object detection than a CPU.

**Comment by NickM-27** [comment]
The models are trained on the same dataset but are different in how they're trained. I'd suggest using a non tiny model and see if it works better for you.
